appcues/src/main/java/com/appcues/ui/extensions/ModifierExt.kt

```diff
@@ -45,28 +44,20 @@ internal fun Modifier.outerPrimitiveStyle(
     gestureProperties: PrimitiveGestureProperties,
     isDark: Boolean,
     defaultBackgroundColor: Color? = null,
+    matchParentBox: BoxScope? = null,
```

Comment:
had to add this as a way to inform that we have a parent (box) we would want to matchParentSize to.